cmd/mpi-operator/app/options/options.go

```diff
@@ -80,4 +81,8 @@ func (s *ServerOption) AddFlags(fs *flag.FlagSet) {
 
 	fs.IntVar(&s.ControllerRateLimit, "controller-queue-rate-limit", 10, "Rate limit of the controller events queue .")
 	fs.IntVar(&s.ControllerBurst, "controller-queue-burst", 100, "Maximum burst of the controller events queue.")
+
+	fs.StringVar(&s.ClusterDomain, "cluster-domain", "", `
```

Comment:
@tenzen-y Do we need this for MPI plugin in KF Trainer ?

Reply:
I don't think so since Trainer does not support `<PodName>.<JobName>.<namespace>.svc` format. The Trainer supports only `<PodName>.<JobName>`.  In case of Trainer, the cluster domain is automatically considered, immediately.